src/Pages/ProductListPage.js

```diff
@@ -4,22 +4,26 @@ import ProductElement from 'Components/HomePage/ProductElement';
 import ProductSkeleton from 'Components/ProductList/ProductSkeleton';
 import Paginator from 'Components/ProductList/Paginator';
 
+import { useFeaturedBanners } from 'utils/hooks/useFeaturedBanners';
+
 import ProductsMock from 'mocks/en-us/products.json';
-import CategoriesMock from '../mocks/en-us/product-categories.json';
 
-const Categories = CategoriesMock.results;
 const allProducts = ProductsMock.results;
 
-const ProductList = () => {
+const ProductListPage = () => {
+    const { data:Categories, isLoading:areCategoriesLoaded } = useFeaturedBanners('category', 30);
 
     const [ chosedCategories, setChosedCategories ] = useState({ items: [] });
     const [ filteredProducts, setFilteredProducts ] = useState({ products: [] });
     const [ hasTimePassed, setHasTimePassed ] = useState(false);
-
-    const filter = () =>         
-        allProducts.filter( (item) => 
+    const [ showResetButton, setShowResetButton] = useState(false);
+    
+    const filter = () => {
```

Comment:
Hard to understand what this filters. Would you mind changing this to a better name description for this function?